examples/serial/board/qemu_arm_virt/serial.system

```diff
@@ -0,0 +1,142 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<system>
+    <memory_region name="uart" size="0x1_000" phys_addr="0x9000000" />
+
+    <memory_region name="tx_data_driver" size="0x200_000" page_size="0x200_000" />
+    <memory_region name="tx_data_client" size="0x200_000" page_size="0x200_000" />
+    <memory_region name="rx_data_driver" size="0x200_000" page_size="0x200_000" />
+    <memory_region name="rx_data_client" size="0x200_000" page_size="0x200_000" />
+
+    <memory_region name="tx_data_client2" size="0x200_000" page_size="0x200_000" />
+    <memory_region name="rx_data_client2" size="0x200_000" page_size="0x200_000" />
+
+    <!-- shared memory for ring buffer mechanism : serial -->
+    <memory_region name="rx_free_serial_driver" size="0x200_000" page_size="0x200_000"/>
```

Comment:
Pretty sure these can be smaller than 1M.

Reply:
Will make an issue and do it in a separate PR.

Reply:
[Done](https://github.com/au-ts/sDDF/issues/66)